Student-Unity-frontend/src/app/components/navigation-bar/navigation-bar.component.ts

```diff
@@ -33,8 +34,15 @@ export class NavigationBarComponent implements OnInit {
     }
 
     logout() {
-        this.signInOutService.signOut();
-        this.router.navigateByUrl('sign-in');
+        this.signInOutService.signOut().subscribe(
+            (res) => {
+                this.router.navigateByUrl('sign-in');
+                this.signInOutService.deleteCookies();
```

Comment:
``this.signInOutService.deleteCookies();`` it is already called inside ``signOut()``

Reply:
I removed it from `signOut()` cause `signOut()` now returns an observable. So what do you think should I delete the cookies before posting the request to the back end or after it?